<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::configurationdialog</class>
 <widget class="QDialog" name="snowgui::configurationdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="remoteconfigurationLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Remote Configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="servicesofferedGroup">
     <property name="title">
      <string>Services offered </string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QCheckBox" name="repositoriesCheckBox">
        <property name="text">
         <string>Repositories</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="devicesCheckBox">
        <property name="text">
         <string>Devices</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="instrumentsCheckBox">
        <property name="text">
         <string>Instruments</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="focusingCheckBox">
        <property name="text">
         <string>Focusing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="guidingCheckBox">
        <property name="text">
         <string>Guiding</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="imagesCheckBox">
        <property name="text">
         <string>Images</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="tasksCheckBox">
        <property name="text">
         <string>Tasks</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="repositoryGroup">
     <property name="title">
      <string>Image Repositories</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="repodbLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Repository Database:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="repodbField"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="repodbButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="snowgui::repositoryconfigurationwidget" name="repositoryconfiguration" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mountGroup">
     <property name="title">
      <string>Mount device</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="mountpointLabel">
        <property name="text">
         <string>Mountpoint:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mountpointField"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="deviceLabel">
        <property name="text">
         <string>Device:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="deviceField"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="mountButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Mount</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="timeBox">
     <property name="title">
      <string>System time</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QLabel" name="sourceTimeLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Source time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="localTimeField">
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="localTimeLabel">
        <property name="text">
         <string>Local time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sourceLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Time Source:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="systemTimeLabel">
        <property name="text">
         <string>System time:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="sourceTimeField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="systemTimeField">
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="syncButton">
        <property name="text">
         <string>Sync</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="sourceBox"/>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QPushButton" name="setButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set System from Source</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="restartWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="restartLabel">
        <property name="text">
         <string>Some Configuration changes require a server restart, but the client application may need to be restarted as well.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="restartButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>Restart</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="localconfigurationLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Local Configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="localconfigurationWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::repositoryconfigurationwidget</class>
   <extends>QWidget</extends>
   <header>repositoryconfigurationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
